File: BOJ/python3/10250.py
'''
ACM 호텔
https://www.acmicpc.net/problem/10250
'''
import sys
import time
start = time.time()  # 시작 시간
sys.stdin = open("input.txt", "r")  # input.txt 파일로 입력

T = int(input())
for t in range(T):
    H, W, N = map(int, input().split())

    r = N % H
    if r != 0:
        c = N // H + 1
    else:
        r = H
        c = N // H

    if c < 10:
        print(f"{r}0{c}")  # 층수, 호수
    else:
        print(f"{r}{c}")

# 현재시각 - 시작시간 = 실행 시간(sec)
print("--------------------")
print(f"Elapsed Time: {round(time.time() - start, 3)}s")

File: BOJ/nodejs/10250.js
/*
ACM 호텔
boj.kr/10250
 */
const start = new Date().getTime();

const fs = require("fs");
const input = fs.readFileSync("input.txt").toString().split("\n");
//const input = fs.readFileSync("/dev/stdin").toString().split("\n");

const T = 1 * input[0];
for (let t = 0; t < T; t++) {
  let [H, W, N] = input[1 + t].split(" ").map(Number);
  let YY, XX;

  N % H === 0 ? (YY = H) : (YY = N % H);
  N % H === 0 ? (XX = Math.floor(N / H)) : (XX = Math.floor(N / H) + 1);

  console.log(`${YY}${XX < 10 ? "0" + XX.toString() : XX}`);
}

const end = new Date().getTime();
console.log("--------------------");
console.log(`Elapsed Time: ${(new Date().getTime() - start) / 1000}s`);
